README.md

```diff
@@ -3,28 +3,79 @@
 [![DeepScan grade](https://deepscan.io/api/teams/14424/projects/24864/branches/769714/badge/grade.svg)](https://deepscan.io/dashboard#view=project\&tid=14424\&pid=24864\&bid=769714)
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/64f466be6e654cd2a2bb790971fb07ef)](https://app.codacy.com/gh/roundhousedesigns/rise-frontend/dashboard?utm_source=gh\&utm_medium=referral\&utm_content=\&utm_campaign=Badge_grade)
 
-## This is the main interface for the RISE Theatre Directory
+## Tech Stack
 
-Current tooling:
-
-*   Node ^19.4.0
+Front end portion of RISE is built using:
 *   Vite
 *   React
 *   GraphQL
 
-## Development
+## Development Setup
+
+### Node Setup
```

Comment:
A bit more detailed assuming a user might not have Node Version Manger experience